app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import { Button } from 'reactstrap'
+import { NavLink } from 'react-router-dom'
+
+class ApartmentIndex extends Component {
+    render(){
+        const { apartments }= this.props
+        return (
+            <div className="body">
+                <h3>Apartments</h3>
+                <br />
+                <article className="cards">
```

Comment:
You have classnames of card and card but no corresponding stylings for those classes.